src/cfnlint/transform.py

```diff
@@ -70,6 +71,13 @@ def _replace_local_codeuri(self):
             if resource_type == 'AWS::Serverless::Function':
 
                 Transform._update_to_s3_uri('CodeUri', resource_dict)
+                auto_publish_alias = resource_dict.get('AutoPublishAlias')
```

Comment:
Is this the only property that allows/requires a fake value for a !Ref? Does it make sense to do this for all properties on a Serverless resource?

Reply:
From the SAM docs.
`
AutoPublishAlias             Ref of a CloudFormation Parameter  Alias resources created by SAM uses a LocicalId <FunctionLogicalId+AliasName>. So SAM either needs a string for alias name, or a Ref to template Parameter that SAM can resolve into a string.
`

Reply:
I don't see any other parameter REFs in https://raw.githubusercontent.com/awslabs/serverless-application-model/master/docs/cloudformation_compatibility.rst